creator-node/src/utils/clusterUtils.ts

```diff
@@ -12,14 +12,8 @@ const config = require('../config')
  */
 class ClusterUtils {
   private _isInitWorker = false
-  private _specialWorkerId = 1
-  get specialWorkerId(): number {
-    return this._specialWorkerId
-  }
-
-  set specialWorkerId(specialWorkerId: number) {
-    this._specialWorkerId = specialWorkerId
-  }
+  private _isSpecialWorker = false
+  private _primaryOnlySpecialWorkerId = -1
```

Comment:
idk what this means haha

Reply:
I was prefixing it to say it's something only the primary uses bc only the primary needs to keep track of which worker ID is the special worker. I reworked this a bit and added docs to try to help but yeah it is unfortunately a hassle to have these variables to track 🙁